src/plexil-adapter/OwInterface.cpp

```diff
@@ -600,9 +560,6 @@ void OwInterface::initialize()
     m_rulSubscriber = new ros::Subscriber
       (m_genericNodeHandle ->
        subscribe("/power_system_node/remaining_useful_life", qsize, rul_callback));
-    m_guardedMoveSubscriber = new ros::Subscriber
-      (m_genericNodeHandle ->
-       subscribe("/guarded_move_result", qsize, guarded_move_callback));
```

Comment:
I don't see where this is set anywhere in the code... better remove it from the header and the destructor

Reply:
Good catch, it was a vestige of the old dummy action client example.